packages/core/src/RenderingEngine/WSIViewport.ts

```diff
@@ -342,11 +362,15 @@ class WSIViewport extends Viewport implements IWSIViewport {
   };
 
   /**
-   * Need to return this as a function to prevent webpack from munging it.
+   * Encapsulate the dicom microscopy fetch so that it can be replaced
+   * with the browser import function.  Webpack munges this and then throws
+   * exceptions trying to get this working, so this has to be provided externally
+   * as a globalThis.browserImportFunction taking the package name, and a set
+   * of options defining how to get the value out of the package.
    */
-  private getImportPath() {
-    return '/dicom-microscopy-viewer/dicomMicroscopyViewer.min.js';
-  }
+  public static getDicomMicroscopyViewer = async () => {
+    return peerImport('dicom-microscopy-viewer');
```

Comment:
@sedghi  - I'm not using the configured peerImport to perform the import, and registering it in the examples.